src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+/**
+ * Parser processes user input and commands.
+ *
+ * @author Jovyn Tan
+ * @version CS2103 AY21/22 Sem 1
+ */
+public interface Parser {
+    /**
+     * Gets the description supplied by the user after a command.
+     * @param command a String that describes the command (e.g. "event")
+     * @param message the String that is input by the user (e.g. "event Meeting /at tomorrow")
+     * @return a String representing the description after the command (e.g. "Meeting /at tomorrow")
+     * @throws DukeException if that the description is invalid
+     */
+    static String getStringFrom(String command, String message) throws DukeException {
+        if (message.length() <= command.length()) {
```

Comment:
Maybe you could store the condition into a variable stating what it is checking for?